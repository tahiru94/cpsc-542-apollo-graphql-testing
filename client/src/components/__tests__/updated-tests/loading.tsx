//andrew
import React from 'react';
import { shallow } from 'enzyme';
import Loading from '../../loading';

describe('Loading',() => {
    it('Renders without error and is defined', () => {
        const wrapper = shallow(<Loading />);
        expect(wrapper).toBeDefined();
    });

    it('Matches snapshot', () => {
        const wrapper = shallow(<Loading />);
        expect(wrapper).toBeDefined();
        expect(wrapper).toMatchSnapshot();
    });

});