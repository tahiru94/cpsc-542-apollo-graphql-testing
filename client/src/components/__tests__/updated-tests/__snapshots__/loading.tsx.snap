// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Matches snapshot 1`] = `ShallowWrapper {}`;
